pygmt/figure.py

```diff
@@ -362,12 +362,17 @@ def show(self, dpi=300, width=500, method=None, waiting=0.5, **kwargs):
             The image resolution (dots per inch) in Jupyter notebooks.
         width : int
             The image width (in pixels) in Jupyter notebooks.
-        method : str
-            How the current figure will be displayed. Options are
+        method : str or None
+            How the current figure will be displayed. Choose from:
 
-            - **external**: PDF preview in an external program [Default]
-            - **notebook**: PNG preview [Default in Jupyter notebooks]
-            - **none**: Disable image preview
+            - ``"external"``: External PDF preview using the default PDF viewer
+            - ``"notebook"``: Inline PNG preview in the current notebook
+            - ``"none"``: Disable image preview
+            - ``None``: Reset to the default display method
+
+            The default display method is ``"external"`` in Python consoles or
+            ``"notebook"`` in Jupyter Notebooks, but can be changed by
+            :func:`pygmt.set_display`.
```

Comment:
Lower-case letters following the discussion in https://github.com/GenericMappingTools/pygmt/pull/2369#discussion_r1108115316.
```suggestion
            The default display method is ``"external"`` in Python consoles or
            ``"notebook"`` in Jupyter notebooks, but can be changed by
            :func:`pygmt.set_display`.
```